src/Parser/Core/Modules/MitigationCheck.js

```diff
@@ -0,0 +1,150 @@
+import React from 'react';
+
+import Analyzer from 'Parser/Core/Analyzer';
+import SPELLS from 'common/SPELLS/index';
+import SpellIcon from 'common/SpellIcon';
+import { formatNumber, formatPercentage } from 'common/format';
+
+import SpellLink from 'common/SpellLink';
+import Enemies from 'Parser/Core/Modules/Enemies';
+
+import {findByBossId} from 'Raids/index';
+import ExpandableStatisticBox from 'Main/ExpandableStatisticBox';
+
+const mitigationBuffs = {'Paladin' : [SPELLS.SHIELD_OF_THE_RIGHTEOUS_BUFF.id,
```

Comment:
Apostrophes can be omitted in object keys

Reply:
We generally don't want to require spec maintainers to have to configure things in the core modules, could you make these constants static properties of the class instead please? Then each spec can make its own version of this module by extending it, and by overriding the prop they can configure the correct spells.

Then you also wouldn't need the class names.